lib/openstax/rescue_from/wrapped_exception.rb

```diff
@@ -0,0 +1,76 @@
+module OpenStax
+  module RescueFrom
+    class WrappedException
```

Comment:
When I first read `WrappedException` I thought of exception causes (when one exception wraps another).  Now that I've read it I get that it is a proxy / decorator for an exception.  This is minor and does not need to change, but you may consider something like `ExceptionProxy` with `@proxy` vars.  Don't worry about it if you are cool with the name.